extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.ts

```diff
@@ -67,11 +67,29 @@ export default class StaticWadoClient extends api.DICOMwebClient {
   }
 
   async searchForSeries(options) {
+    // asking to include Series Date, Series Time, Series Description
+    // and Series Number in the series metadata returned to better sort series
+    // in preLoad function
+    const commaSeparatedFields = [
+      '00080021',
+      '00080031',
+      '0008103E',
+      '00200011',
+    ].join(',');
+
+    const queryParamsMinimun = {
+      includefield: commaSeparatedFields,
+    };
+    const updatedOptions = {
+      ...options,
```

Comment:
What happens if options already has a query params?  You need to join this. 
Also, the include field should only be applied at the required level.
Note that static wado client is used for more back ends than just static wado ones.  It also sometimes supports real queries, so I wouldn't just delete the remaining queries.